### PR TITLE
New package: Microsoft.MSKLC version 1.4

### PR DESCRIPTION
**Pull request by noraa-junker**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/122068)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

The installer appears to be behind the package manager window.

**Comment by stephengillie** [comment]
image

**Comment by stephengillie** [comment]
Hi @Aaron-Junker,

The package didn't install silently. Is an installer switch missing?

File: manifests/m/Microsoft/MSKLC/1.4/Microsoft.MSKLC.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.MSKLC
```
Comment (Trenly) [inline manifests/m/Microsoft/MSKLC/1.4/Microsoft.MSKLC.installer.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.KeyboardLayoutCreator
```

File: manifests/m/Microsoft/MSKLC/1.4/Microsoft.MSKLC.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.MSKLC
```
Comment (Trenly) [inline manifests/m/Microsoft/MSKLC/1.4/Microsoft.MSKLC.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.KeyboardLayoutCreator
```

File: manifests/m/Microsoft/MSKLC/1.4/Microsoft.MSKLC.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.MSKLC
```
Comment (Trenly) [inline manifests/m/Microsoft/MSKLC/1.4/Microsoft.MSKLC.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.KeyboardLayoutCreator
```

File: manifests/m/Microsoft/MSKLC/1.4/Microsoft.MSKLC.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.MSKLC
+PackageVersion: "1.4"
+InstallerType: exe
+Installers:
+- InstallerUrl: https://download.microsoft.com/download/6/f/5/6f5ce43a-e892-4fd1-b9a6-1a0cbb64e6e2/MSKLC.exe
```
Comment (Trenly) [inline manifests/m/Microsoft/MSKLC/1.4/Microsoft.MSKLC.installer.yaml:8]: This appears to be a self-extracting archive which requires user interaction to install. Do you know if there are any silent switches?

Comment (Trenly) [inline manifests/m/Microsoft/MSKLC/1.4/Microsoft.MSKLC.installer.yaml:8]: This package also has a dependency on .NET Framework 3.5

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Aaron-Junker** [comment]
With the self extracting archive it's too complicated to create a package.

**Pull request closed without merging**